core/src/main/java/nl/nn/adapterframework/jdbc/JdbcTransactionalStorage.java

```diff
@@ -1013,9 +1013,10 @@ public boolean isActive() {
 		return active;
 	}
 
+	@Override
 	@IbisDoc({"the name of the column that stores the hostname of the server", "host"})
 	public void setHostField(String hostField) {
-		this.hostField = hostField;
+		super.setHostField(hostField);
 	}
 
```

Comment:
Mag dan weg toch? als het toch niets doet...

Reply:
toch niet, want de super is protected